TACC/Core-Styles
# fix: checkbox feild layout broken by form--login

## Overview

Fix checkbox field being center-aligned on a login form.

## Related

- required by https://github.com/tapis-project/authenticator/

## Changes

- **changed** selector to not match checkbox fields

## Testing

1. Add a single checkbox field in a login form.
2. Verify checkbox and label are left-aligned.

## UI

| before (during in situ test) | after (approx.) (during in situ test) |
| - | - |
| <img width="1125" alt="after (during situ test)" src="https://github.com/TACC/Core-Styles/assets/62723358/b0bbcaca-815c-4951-8a6a-492b1cc3526c"> | <img width="1125" alt="before (during situ test)" src="https://github.com/TACC/Core-Styles/assets/62723358/708fb88a-f031-41b4-9da6-2444ba51f4d5"> |
